bin/ch-fromhost

```diff
@@ -53,17 +63,19 @@ Which files to inject (one or more required; can be repeated):
   -c, --cmd CMD    listed in the stdout of CMD
   -f, --file FILE  listed in file FILE
   -p, --path PATH  inject the file at PATH
-  --cray-mpi       Cray-enable MPICH/OpenMPI installed within the image
-  --nvidia         recommended by nVidia (via "nvidia-container-cli list")
+  -o, --ofi  PATH  add host compiled OFI DSO provider(s) to image libfabric
+  --host-ofi       ofi providers at path CH_FROMHOST_OFI
```

Comment:
Will the term OFI be familiar to users?

Reply:
Hard to say. I didn't know libfabric was OFI. Some alternatives would be:

- `--libfabric`, or
- `--libfabric-dl` (when building loadable shared objects with libfabric you add `dl`, e.g., `--with-gni=dl`

Reply:
Maybe talk with Hunter and/or Tim about it?